src/py21cmfast/wrapper.py

```diff
@@ -338,18 +365,24 @@ class AstroParams(StructWithDefaults):
         but is required by this class to set some default behaviour.
     HII_EFF_FACTOR : float, optional
     F_STAR10 : float, optional
+    F_STAR7_MINI : float, optional
```

Comment:
Here could go the explanation of why it's 7 vs 10 for F_STAR.

Reply:
1e10 is a typical value for atomic cooling halos while 1e7 is typical for minihalos. we don't talk about 1e7 atomic cooling halos or 1e10 minihalos.